scripts/dependabot_generate_changesets.ts

```diff
@@ -0,0 +1,93 @@
+import fsp from 'fs/promises';
```

Comment:
I'd rename this script to `dependabot_handle_version_update` or something like this.
It's doing a bit more than generating changeset (i.e. it's also pushing it to repo).


Also - consider writing a happy case test case for this. 
Something like https://github.com/aws-amplify/amplify-backend/blob/main/scripts/components/release_lifecycle.test.ts .